py3status/modules/dpms.py

```diff
@@ -7,6 +7,8 @@
 by clicking on 'DPMS' in the status bar.
 
 Configuration parameters:
+    button_toggle: mouse button to toggle DPMS (default 1)
+    cache_timeout: refresh interval for this module (default 60)
```

Comment:
> Calling xset -q | grep -iq ... (two things) every 5 seconds? I don't want xrandr module cause screen freeze for ~0.5 second every time it refreshes issue for dpms.

I don't think they are related.  that is a xrandr specific issue.  how about 15?